index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>PEGS 2023</title>
+  <link rel="icon" type="image/x-icon" href="/images/favicon.png">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+  <link rel="stylesheet" href="Style.css">
+  <script src="script.js" defer></script>
+</head>
+
+<body>
+  <div class="first-page">
+    <!-- headder -->
+    <header class="headder">
+      <a id="hamburger" class="hamburger text-dark">
+        <i class="glyphicon glyphicon-menu-hamburger mt-4"></i>
+      </a>
+      <div class="headding">
+        <div class="contact">
+          <i class="material-icons text-light">facebook</i>
+          <div class="text-light">English</div>
+          <div class="text-light">My Page</div>
+          <div class="text-light">Logout</div>
+        </div>
+        <div class="logo">
+          <div>
+            <i class="material-icons">school</i>
+            <h2>PE Global Summit</h2>
+          </div>
+          <ul class="logo-menu">
+            <a href="about.html">
+              <li>About</li>
+            </a>
+            <a href="#main-program">
+              <li>Program</li>
+            </a>
+            <a href="#">
+              <li>Join</li>
+            </a>
+            <a href="#">
+              <li>Sponsor</li>
+            </a>
+            <a href="#">
+              <li>News</li>
+            </a>
+            <a href="#">
+              <li>PEGS Campaign</li>
+            </a>
+          </ul>
+        </div>
+      </div>
```

Comment:
### NAV FOR HOME AND ABOUT PAGE

- Hey, to enforce a better use of the semantic tag, you should wrap this content in a nav tag as this is the introductory content of the page and the nav tag is used as a container holding them. You could make the nav 2 because you have to separate the nav. please let's adjust this part of our HTML structure.